examples/go/with_proto/with_proto.bazelproject

```diff
@@ -11,5 +11,5 @@ gazelle_target: //:gazelle
 import_run_configurations:
   # Test that validates that macro expansion works, as per issue:
   #    https://github.com/bazelbuild/intellij/issues/4112#event-7958662669
-  # This configuration should be executed on both 'run' (the green arrow) and 'debug' (the bug) modes.
+  # This configuration should only be executed on both 'run' (the green arrow) mode.
```

Comment:
I should have written a better comment, sorry about that.
The main issue is that, because of how we pass env vars to debuggers, the IntelliJ macros (e.g. `$ProjectName$`) are not expanded. So, the env vars get passed, but the macros don't get expanded.

I've added a new run configuration to test env vars, which works on both run and debug.